GameData/KerbalismConfig/Support/ROCapsules.cfg

```
@@ -548,29 +937,40 @@
 		%max_pressure = 0.35
 	}
 }
+@PART[ROC-D2MissionModule2]:NEEDS[ROCapsules,FeatureHabitat]:AFTER[RO-Kerbalism]
+{
+    @MODULE[Habitat]
+	{	
+		volume = 5.5        //guesstimate
+		surface = 4.347         //m^2
+		max_pressure = 0.35
+	}
+}
 @PART[ROC-ApolloCM]:NEEDS[ROCapsules,FeatureHabitat]:AFTER[zzzKerbalism]
 {
 	@MODULE[Habitat]
 	{	
-		%volume = 5.9
-		%surface = 15.71	 //guesstimate
+		%volume = 6.2	//m^3
```

```suggestion
		%volume = 5.9	//m^3
```
Previous value was sourced, 6.2 is accurate for the proposed block III variant, but not for the actual Apollo CM.